pyabc/sampler/redis_eps/sampler.py

```diff
@@ -169,12 +176,14 @@ def __init__(self,
                  batch_size: int = 1,
                  look_ahead: bool = False,
                  look_ahead_delay_evaluation: bool = True,
+                 max_n_eval_look_ahead_factor: float = 10.,
```

Comment:
Wouldnt it maybe be better to default it to infinity, since this is kind of only useful when memory is an issue? 
Also, if we ever want to enable the buffering for several generations ahead.

Reply:
Not sure. I think this may also be an issue with the short-running models, where we basically had a few thousand evaluations in round two, all of which were then accepted nearly, at a population size is 100 maybe. This was there because the time between iterations was so long. Then checking all acceptances may again take some time, so that setting it to inf may indeed result in longer times than a reasonable limit. Not sure if 10 is too aggressive? Are there cases where acceptance rate drops by a factor of 10 quickly? Then one could use a larger default ...